operations/tempo-mixin-compiled/dashboards/tempo-writes.json

```diff
@@ -1460,6 +1460,196 @@
      "renderer": "flot",
      "seriesOverrides": [
 
+     ],
+     "spaceLength": 10,
+     "span": 6,
+     "stack": true,
+     "steppedLine": false,
+     "targets": [
+      {
+       "expr": "sum by (status) (\n  label_replace(label_replace(rate(tempo_live_store_complete_queue_length{cluster=~\"$cluster\", pod=~\"live-store-zone.*\"}[$__rate_interval]),\n  \"status\", \"${1}xx\", \"status_code\", \"([0-9])..\"),\n  \"status\", \"${1}\", \"status_code\", \"([a-zA-Z]+)\"))\n",
+       "format": "time_series",
+       "interval": "1m",
+       "legendFormat": "{{status}}",
+       "refId": "A"
+      }
+     ],
+     "thresholds": [
+
+     ],
+     "timeFrom": null,
+     "timeShift": null,
+     "title": "Pending Queue Length",
```

Comment:
Does this belong to the tempo-writes dashboard? Live store is now in the read path